src/test/java/seedu/algobase/logic/commands/DefaultModelStub.java

```diff
@@ -84,11 +83,36 @@ public void setProblem(Problem target, Problem editedProblem) {
         throw new AssertionError("This method should not be called.");
     }
 
+    @Override
+    public void updateFilteredPlanList(Predicate<Plan> predicate) {
+
```

Comment:
Add `throw new AssertionError("This method should not be called.");` in the method body to be consistent with the rest.